README.md

```diff
@@ -27,54 +27,54 @@ IPFS in Node is a work in progress. As such, there's a few things you can do rig
+      - [x] Identify Protocol [identify](https://github.com/diasdavid/js-libp2p-swarm/tree/master/src/identify).
+      - [x] [js-ipfs-ping](https://github.com/diasdavid/js-ipfs-ping).
       - [x] Connection Interface [abstract-connection](https://github.com/diasdavid/abstract-connection)
       - Transports
         - [x] Transport Interface [abstract-transport](https://github.com/diasdavid/abstract-transport)
-        - [x] [libp2p-tcp](https://github.com/diasdavid/node-libp2p-tcp)
-        - [ ] [libp2p-udp](https://github.com/diasdavid/node-libp2p-udp)
-        - [ ] [libp2p-udt](https://github.com/diasdavid/node-libp2p-udt)
-        - [ ] [libp2p-utp](https://github.com/diasdavid/node-libp2p-utp)
+        - [x] [libp2p-tcp](https://github.com/diasdavid/js-libp2p-tcp)
+        - [ ] [libp2p-udp](https://github.com/diasdavid/js-libp2p-udp)
+        - [ ] [libp2p-udt](https://github.com/diasdavid/js-libp2p-udt)
+        - [ ] [libp2p-utp](https://github.com/diasdavid/js-libp2p-utp)
         - [ ] libp2p-webrtc
         - [ ] libp2p-cjdns
       - Upgrades
         - [ ] libp2p-tls
       - Stream Muxing
         - [x] [abstract-stream-muxer](https://github.com/diasdavid/abstract-stream-muxer).
-        - [x] [node-spdy-stream-muxer](https://github.com/diasdavid/node-spdy-stream-muxer) _stream muxer_. [Discussion issue](https://github.com/ipfs/node-ipfs/issues/23).
-        - [x] [libp2p-spdy](https://github.com/diasdavid/node-libp2p-spdy/blob/master/src/index.js) _stream muxer upgrade_
+        - [x] [js-spdy-stream-muxer](https://github.com/diasdavid/js-spdy-stream-muxer) _stream muxer_. [Discussion issue](https://github.com/ipfs/js-ipfs/issues/23).
+        - [x] [libp2p-spdy](https://github.com/diasdavid/js-libp2p-spdy/blob/master/src/index.js) _stream muxer upgrade_
       - Protocol Muxing
-        - [x] [node-multistream](https://github.com/diasdavid/node-multistream) _protocol muxer_. [Discussion issue](https://github.com/ipfs/node-ipfs/issues/24).
-        - [x] [node-multistream](https://github.com/diasdavid/node-multistream).
-    - [ ] Distributed Record Store. [Discussion issue](https://github.com/ipfs/node-ipfs/issues/25).
-      - [x] [node-ipfs-record](https://github.com/diasdavid/node-ipfs-record) _record (needs MerkleDAG node)_.
-      - [x] [node-ipfs-distributed-record-store](https://github.com/diasdavid/node-ipfs-distributed-record-store).
-      - [x] [node-ipfs-kad-record-store](https://github.com/diasdavid/node-ipfs-kad-record-store) _implements abstract record store_.
+        - [x] [js-multistream](https://github.com/diasdavid/js-multistream) _protocol muxer_. [Discussion issue](https://github.com/ipfs/js-ipfs/issues/24).
+        - [x] [js-multistream](https://github.com/diasdavid/js-multistream).
+    - [ ] Distributed Record Store. [Discussion issue](https://github.com/ipfs/js-ipfs/issues/25).
+      - [x] [js-ipfs-record](https://github.com/diasdavid/js-ipfs-record) _record (needs MerkleDAG node)_.
```

Comment:
Ah, hence the rename to js makes confusions like this less likely to happen :+1: